agent/xds/testdata/endpoints/terminating-gateway-default-service-subset.golden

```diff
@@ -1,40 +1,6 @@
 {
   "versionInfo": "00000001",
   "resources": [
-    {
-      "@type": "type.googleapis.com/envoy.api.v2.ClusterLoadAssignment",
-      "clusterName": "api.default.dc1.internal.11111111-2222-3333-4444-555555555555.consul",
-      "endpoints": [
-        {
-          "lbEndpoints": [
-            {
-              "endpoint": {
-                "address": {
-                  "socketAddress": {
-                    "address": "10.10.1.1",
-                    "portValue": 8081
-                  }
-                }
-              },
-              "healthStatus": "HEALTHY",
-              "loadBalancingWeight": 1
-            },
-            {
-              "endpoint": {
-                "address": {
-                  "socketAddress": {
-                    "address": "10.10.1.2",
-                    "portValue": 8081
-                  }
-                }
-              },
-              "healthStatus": "HEALTHY",
-              "loadBalancingWeight": 1
-            }
-          ]
-        }
-      ]
-    },
```

Comment:
I didn't follow the test fixture change in enough detail but this is expected right? That existing tests that didn't use hostnames should have their results change?

Reply:
Yep, this is expected. 

I updated the API service so that it has a mix of hostnames/IPs rather than just IPs here:
https://github.com/hashicorp/consul/pull/7999/files#diff-f8966ffd7dc2c27418ed8181bcedeecdR1620

Alternatively I could add a third service rather than modify one of the existing ones.